runtime/syntax/karel.vim

```diff
@@ -0,0 +1,116 @@
+" Vim syntax file
+" Language:    KAREL
+" Last Change: 2024-11-17
+" Maintainer:  Kirill Morozov <kirill@robotix.pro>
+" Credits:     Jay Strybis for the initial implementation and Patrick Knosowski
+"              for a couple of fixes.
+
+if exists("b:current_syntax")
+  finish
+endif
+
+if version < 600
+  syntax clear
+endif
```

Comment:
I don't think we need this block anymore.

Reply:
Done.